ci-operator/step-registry/redhat-chaos/krkn-hub-prometheus-tests/redhat-chaos-krkn-hub-prometheus-tests-chain.yaml

```diff
@@ -0,0 +1,17 @@
+
+chain:
+  as: redhat-chaos-krkn-hub-prometheus-tests
+  env:
+  - name: GC
+    default: "False"
+  - name: ITERATION_MULTIPLIER_ENV
+    default: "4"
+  - name: CHURN
+    default: "False"
+  - name: PROFILE_TYPE
+    default: "reporting"
+  steps:
+  - ref: openshift-qe-cluster-density-v2
+  - ref: redhat-chaos-pod-scenarios-prometheus-disruption
```

Comment:
We have a pvc prometheus test as well I believe

Reply:
Right! Added, thanks.